React/CxxBridge/JSCExecutorFactory.mm

```diff
@@ -25,6 +25,13 @@
         [NSString stringWithUTF8String:message.c_str()]);
     };
     react::bindNativeLogger(runtime, iosLoggingBinder);
+
+    react::PerformanceNow iosPerformanceNowBinder =[]() {
+        // CACurrentMediaTime() returns the current absolute time, in seconds
+        return CACurrentMediaTime() * 1000;
+    };
+    react::bindNativePerformanceNow(runtime, iosPerformanceNowBinder);
```

Comment:
Is there a reason to bind this here (in each executor factory) instead of in somewhere more central like Instance::bindBridge? (I guess that would probably involve changing the API for bindBridge to add another parameter, which would be a breaking change for any other executors that aren't part of RN core). I'm just wondering because it seems like the difference between how we bind nativePerformanceNow is based on the platform we're running on, not the JS VM, so we don't necessarily need this to be in the JS executor factory.

Reply:
It seemed like a good place to use without making any breaking changes. Logging binder is there too.